### PR TITLE
Updated the maven repository with maven dependency inclusion

### PR DESCRIPTION
**Pull request by akilagithub**
As per maven latest updates added the dependencies to build the project successfully. pom.xml is modfiied with additional repositories

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -51,10 +51,38 @@
...
+                    <name>Central Repository</name>
+                    <url>https://repo.maven.apache.org/maven2</url>
+             </pluginRepository>
+       </pluginRepositories>
+
 
     <dependencyManagement>
         <!-- Import the maven Spring IO Platform Bill Of Materials (BOM) -->
+
+
```
Comment (SachinKSingh28) [inline pom.xml]: Could you please remove these extra lines?

File: pom.xml [hunk 2]
```diff
@@ -314,7 +342,9 @@
             <groupId>com.googlecode.json-simple</groupId>
             <artifactId>json-simple</artifactId>
         </dependency>
-
+		
+		
+
```
Comment (SachinKSingh28) [inline pom.xml]: Could you please remove these extra lines?

File: pom.xml [hunk 3]
```diff
@@ -343,6 +373,8 @@
             </testResource>
         </testResources>
         <plugins>
+		
+
```
Comment (SachinKSingh28) [inline pom.xml]: Could you please remove these extra lines?

**Review by SachinKSingh28 — CHANGES_REQUESTED** [review]
Thanks @akilagithub for the PR. Requesting few minor changes. In the mean time, I'll check if the repository is building fine.

File: pom.xml (hunk 1 above)
Comment (akilagithub) [reply pom.xml]: removed and updated in my repository

File: pom.xml (hunk 2 above)
Comment (akilagithub) [reply pom.xml]: removed and updated in my repository

File: pom.xml (hunk 3 above)
Comment (akilagithub) [reply pom.xml]: removed and updated in my repository

**Comment by akilagithub** [comment]
Any updates on this PR?

**Comment by SachinKSingh28** [comment]
@akilagithub Looks good. could you please squash the commits?

**Comment by akilagithub** [comment]
I have squsashed commit

**Comment by akilagithub** [comment]
> I have squashed commit

**Review by SachinKSingh28 — APPROVED** [review]

**Comment by akilagithub** [comment]
Thank you Sachin

**Pull request merged**